cloudify_aws/s3/resources/bucket.py

```diff
@@ -109,7 +123,25 @@ def create(ctx, iface, resource_config, params, **_):
             del params['CreateBucketConfiguration']
 
     # Actually create the resource
-    bucket = iface.create(params)
+    try:
+        bucket = iface.create(params)
+    except BaseException as e:
+        ctx.logger.error(str(e))
```

Comment:
I do not think that we need to log the exception.

We do need to check here that the error message indicating that ACL is the reason that this failed. For example
```
except BaseException as e:
   if 'the ACL is the problem' in str(e):
      # Then do the put_public_access_block thing.
```